client/js/socket.js

```diff
@@ -11,6 +11,10 @@ const socket = io({
 	reconnection: !$(document.body).hasClass("public"),
 });
 
+$("#connection-error").on("click", function() {
+	$(this).removeClass("shown");
+});
```

Comment:
I'm not sure to understand what this is for. My understanding is that the banner is now clickable and clicking it removes it. What's the reason behind it?

Reply:
Upload errors re-use that red banner.